pkg/clients/elasticache/elasticache.go

```diff
@@ -219,9 +220,21 @@ func automaticFailoverEnabled(af elasticache.AutomaticFailoverStatus) *bool {
 	return &r
 }
 
+func versionMatches(kubeVersion *string, awsVersion *string) bool {
```

Comment:
What do you think about using `switch` instead of three separate conditions in this function? I think `switch` makes it easier to see what's going on when ordered if's are needed.

Reply:
fixed